.travis.yml

```diff
@@ -14,6 +14,7 @@ before_install:
   - go get -t -v ./...
 
 script:
+  - diff -u <(echo -n) <(gofmt -d -s .)
```

Comment:
Shouldn't this also be diff-ing with "gofmt -d -s ./..." ?

I can see this failing builds and confusing people because they don't understand why. Can we put this in a script with a clear message that the build is failing due to a gofmt issue?